database/seeds/PoemSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class PoemSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('poems')->insert([
+            'title' => Str::random(10),
+            'body' => Str::random(100),
+            'created_at' => Carbon::now()->format('Y-m-d H:i:s'),
+            'updated_at' => Carbon::now()->format('Y-m-d H:i:s')
+        ]);
```

Comment:
Typically, you want the seeder to use the factory to create multiple records. I would do that here.

https://laravel.com/docs/7.x/seeding#using-model-factories